third-party/dms-guild/classes/class-pale-master.xml

```diff
@@ -0,0 +1,1891 @@
+			<p class="indent">In addition, you use your Intelligence modifier when setting the saving throw DC for a pale master spell you cast and when making an attack roll with one.</p>
+			<center>
+				<p><strong>Spell save DC</strong> = 8 + your proficiency bonus + your Intelligence modifier</p>
+				<p><strong>Spell attack modifier</strong> = your proficiency bonus + your Intelligence modifier</p>
+			</center>
+			<h5>RITUAL CASTING</h5>
+			<p>You can cast a necromancy spell as a ritual if that spell has the ritual tag, as the spell is in your grimoire. You don't need to have the spell prepared.</p>
+			<h5>SPELLCASTING FOCUS</h5>
+			<p>You can use an arcane focus (found in chapter 5) as a spellcasting focus for your pale master spells. Your <i>Undead Graft</i>, obtained at 3rd level, becomes your spellcasting focus.</p>
+			<h5>LEARNING SPELLS OF 1ST LEVEL AND HIGHER</h5>
+			<p>Each time you gain a pale master level, you are able to decipher more of your grimoire. When you gain a new spell slot level, you are able to understand and cast the corresponding level of spells if you have them prepared.</p>
+			<h5>REPLACING A GRIMOIRE</h5>
+			<p>Your grimoire is an ancient tome that was given to you and is thus practically irreplaceable.</p>
+			<p class="indent">However, you are able to copy a spell from your own grimoire into another tome - for example, if you want to make a backup copy of your grimoire. This is a lengthy and painstaking process, particularly for the spells you have not yet deciphered and used.</p>
+			<p class="indent">For spells you are able to cast, coping the spell is just like scribing onto a scroll, but faster and easier, since you understand the notation and already know how to cast the spell. You need spend only 1 hour and 10 gp. for each level of the copied spell.</p>
+			<p class="indent">For spells you cannot yet cast, the process is much more difficult. To retain the incantation's meaning, you must copy it exactly. To do so, you may make an arcana check against a DC 15 for second levels spells. This DC further increases by 2 for every additional spell level you wish to transcribe.</p>
+			<p class="indent">For this reason, it is very rare for a pale master to make copies of their grimoire; rather choosing to value it above all their other material possessions.</p>
+			<h5>THE GRIMOIRE’S APPEARANCE</h5>
+			<p>A pale master's grimoire tends to don runes inlaid into a black leather cover, which symbolize the dark magic contained within.</p>
+		</description>
+		<sheet display="false">
+			<description>You can prepare {{pale master:spellcasting:prepare}} spells from the your spellbook. You use an arcane focus to cast spells. and can also cast pale master spells as a ritual.</description>
+		</sheet>
+		<spellcasting name="Pale Master" ability="Intelligence" prepare="true">
+			<list known="true">Pale Master</list>

```

Comment:
can i ask why is it preferable to use last name/part of the publisher's name as first part of that support? I'm really confused by the logic, wouldn't it be better to just use abbreviation of the source?
Also another question, is there a point to this way of naming supports other than making it unique for possible feature additions with the same/similar names?

Reply:
It's just prefixed `GRIM_` as in GRIM press for author abbreviation due to usage across multiple different sources. Like `HUFF_` makes it easier to pinpoint where it comes from within the DMSG.

Unique and 'grouped' due to abbreviation.

Reply:
Grim Press isn't an author, it's a "publisher", that's why i'm confused. I still can't see logic in this way of naming supports, but either way, since Pale Master got re-branded as Grave Warden and i already changed it to fit that, would you prefer doing `DMSG_GPGW_Grave_Warden` or `DMSG_GRIM_Grave_Warden`?